README.md

```diff
@@ -0,0 +1,120 @@
+# Robocrystallographer
+
+[![PyPI version](https://badge.fury.io/py/robocrys.svg)](https://pypi.org/project/robocrys/)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/47f851408d364efa9a8cdf0ed844cd8b)](https://www.codacy.com/app/utf/robocrystallographer)
+[![Codacy Badge](https://api.codacy.com/project/badge/Coverage/47f851408d364efa9a8cdf0ed844cd8b)](https://www.codacy.com/app/utf/robocrystallographer)
+[![CircleCI](https://circleci.com/gh/hackingmaterials/robocrystallographer.svg?style=svg)](https://circleci.com/gh/hackingmaterials/robocrystallographer)
+
+
+Robocrystallographer is a tool to generate text descriptions of crystal
+structures. Similar to how a real-life crystallographer would analyse a
+structure, robocrystallographer looks at the symmetry, local environment, and
+extended connectivity when generating a description. The package includes
+utilities for identifying molecule names, component orientations,
+heterostructure information, and more...
+
+## Usage
+
+Robocrystallographer can be used from the command-line or from a python API.
+The package integrates with the [Materials Project](https://materialsproject.org)
+to for allow generation of structure descriptions directly from Materials Project
+ids. For example, to generate the description of SnO<sub>2</sub> 
+([mp-856](https://materialsproject.org/materials/mp-856/)), one
+can simply run:
+
+```bash
+robocrys mp-856
+```
+
+Alternatively, a structure file can be specified in place of a Materials Project id.
+Robocrystallographer supports the same file formats as 
+[pymatgen](http://pymatgen.org), including the Crystallographic Information 
+Format (CIF), and common electronic structure package formats such as POSCAR files.
+More information can be found on the 
+[command-line interface page](https://hackingmaterials.github.io/robocrystallographer/cli.html).
+
+### Python interface
+
+The two core classes in robocrystallographer are:
+
+- `StructureCondenser`: to condense the structure into an intermediate JSON
+  representation.
+- `StructureDescriber`: to turn the condensed structure into a text description.
+
+A minimal working example for generating text descriptions is simply:
+
+```python
+from robocrys import StructureCondenser, StructureDescriber
+
+condenser = StructureCondenser()
+describer = StructureDescriber()
+
+condensed_structure = condenser.condense_structure(structure)
+description = describer.describe(condensed_structure)
+```
+
+Where `structure` is a pymatgen Structure object. Both classes have many
+options for customising the output of the structure
+descriptions. More information is provided in the 
+[module documentation](https://hackingmaterials.github.io/robocrystallographer/modules).
+
+### Intermediate JSON format
+
+The format of the intermediate JSON representation is detailed on the
+[condensed structure format page](https://hackingmaterials.github.io/robocrystallographer/format.html).
+
+
```

Comment:
You should think more of "other" things to do with intermediate (or descriptive) JSON format.

e.g.
JSON -> simple list of tags e.g. ["layered", "corner-sharing", "octahedra", "rutile"]. e.g. for MP web site people can just type in some keywords that they want in a structure
JSON -> Mongo-queryable format